src/makielayout/blocks.jl

```diff
@@ -94,7 +94,7 @@ macro Block(name::Symbol, body::Expr = Expr(:block))
     esc(q)
 end
 
-_defaultstring(x) = string(x)
+_defaultstring(x) = string(MacroTools.striplines(x))
```

Comment:
Could it work to do this without MacroTools, by just having:
```julia
striplines(@nospecialize(expr)) = expr
function striplines(expr::Expr)
    args = [striplines(elem) for elem in expr.args if !(elem isa LineNumberNode)]
    return Expr(expr.head, args...)
end

```
For a simple test, this seems to work fine!

Reply:
yeah sure, was just to lazy to copy paste and thought MacroTools is always an indirect dep anyway